File: Recursion/Multiplication of number/By iterative.cpp
// including the header file
#include <iostream>

// using the namespace
using namespace std;

// main function of the program
int main(){
	
	// initialize the variable
	int m, n, prod=0;
	
	// getting input from user
	cin>>m>>n;
	
	// loop runs for "1" to "n"
	for(int i = 1; i<=n; i++){
		
		// suming the product = product + m;
		prod += m;
		
	}
	
	// print product
	cout<<prod;
}

File: Recursion/Multiplication of number/By iterative.java
import java.util.*;

class HelloWorld {
    
    public static void main(String[] args) {
        // initialize the variable
	    int m, n, prod=1;
	    
	    // getting input from user
	    Scanner s = new Scanner(System.in);
	    m = s.nextInt();
	    n = s.nextInt();
	    // loop runs for "1" to "n"
	    for(int i = 1; i<=n; i++){
	    	
	    	// suming the product = product + m;
	    	prod += m;
	    	
	    }
	    
	    System.out.println(prod);
    }
}
